scripts/common/preflights.sh

```diff
@@ -411,6 +411,18 @@ function host_preflights() {
         opts="${opts} --is-upgrade"
     fi
 
+    # Remove previous file if it exists
+    if [ -f "$VENDOR_PREFLIGHT_SPEC" ]; then
```

Comment:
You should use curly braces in your expressions like so, both for consistency and by not doing so it can lead to subtle problems in variable expansion with spaces, arrays etc `"${VENDOR_PREFLIGHT_SPEC}"`